betty/plugin/__init__.py

```diff
@@ -316,6 +316,32 @@ def depends_on(self) -> Set[MachineName]:
 )
 
 
+class ExtensionDependentPluginDefinition(PluginDefinition):
```

Comment:
Now that we want to introduce `@required_extension`, we no longer need to change anything about definitions.